README.md

```diff
@@ -57,8 +57,17 @@ Also it should not depend on any of the variables defined in `tests/playbooks/va
 To run the tests, you can either call `make test`, or `make test_<playbook_name>` to only run a specific one.
 To perform codestyle linting and ansible sanity checks, run `make lint sanity`.
 
-To (re-)record tests, you first need to setup a pulp instance ([pulplift](https://github.com/pulp/pulplift) is recommended here).
-With it's connection details configured in `tests/playbooks/vars/server.yaml`, you can run `make record_<playbook_name>`.
+To (re-)record tests or run live tests, you first need to setup a pulp
+instance, with it's connection details configured in
+`tests/playbooks/vars/server.yaml`. The `tests/run_container.sh` script is
+provided and allows you to run a command with a [Pulp in
+one](https://pulpproject.org/pulp-in-one-container/) container active. It
+requires Docker or Podman to be installed. For example:
```

Comment:
I think you should mention, that you still need to have the `server.yaml` point in the right direction (especially, if this method is described, after the user tried to fiddle around with it for a vm). Would you mind replacing the dead reference to pulplift with mentioning the vagrant boxes in pulp_installer?

Also in this repo I prefer the on sentence per line docs.